crowbot/backend/management/commands/fill_question_database.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from backend.models import Course, Question, Answer
+from backend.lemmalize import lemmalize
+import pickle
+
+class Command(BaseCommand):
+    help = 'Fills the database with often asked questions for all courses.'
+
+    def handle(self, *args, **options):
+        # feel free til å forbedre formuleringen til sprøsmålene og legge til flere (husk da å legge til i listen
+        # list_of_questions)!
```

Comment:
For å være sikker på at alle elementene blir med i lista, kan man bare definere dem i lista:
```
list_of_questions = [
  'How many exercises are mandatory',
  'If I have completed the exercises in previous years, are they still valid',
  # osv.
]
```